ngraph/core/src/op/interpolate.cpp

```diff
@@ -175,6 +175,18 @@ std::vector<int64_t> op::v4::Interpolate::get_axes() const
 
 static constexpr float epsilon = 1.0e-6f;
 
+namespace
+{
+    int64_t multiply_bound_and_scale(int64_t bound, float scale)
+    {
+        if (bound == -1)
```

Comment:
Can the bound be less than -1?

Reply:
No. According to https://github.com/openvinotoolkit/openvino/blob/master/ngraph/core/src/dimension.cpp, the value of upper bound can be non-negative integer or `-1`, and `-1` is a special value indicating "infinity".